openstack_plugin/resources/compute/host_aggregate.py

```diff
@@ -47,16 +47,21 @@ def _add_hosts(openstack_resource, hosts):
     ctx.instance.runtime_properties['hosts'] = hosts
 
 
-def _remove_hosts(openstack_resource, hosts):
+def _remove_hosts(openstack_resource, hosts, update_on_remove=False):
```

Comment:
@0lvin What is the case you need to add this Flag ? `update_on_remove`

Reply:
I need flag for cleanup `runtime_properties` after each action with `openstack(openstack_resource.remove_host(host)`. It's part of support `cancel`/`failure` on manager.